commands/request.go

```diff
@@ -28,8 +28,9 @@ type Context struct {
 	config     *config.Config
 	LoadConfig func(path string) (*config.Config, error)
 
-	node          *core.IpfsNode
-	ConstructNode func() (*core.IpfsNode, error)
+	node                     *core.IpfsNode
+	ConstructNode            func() (*core.IpfsNode, error)
+	ContextIsReadyToBeClosed chan bool
```

Comment:
let's use `InitDone` as mentioned above. 

style note: `context.ContextIsReadyToBeClosed` is not very go idiomatic, something shorter + less stuttered is typical